docs/dev/generator.js

```diff
@@ -536,6 +541,17 @@ const processMd = async ({baseConfig, relDir, in_folder, iPath, oPath}) => {
   theader.remove();
 
   // Process code snippets.
+  const linkifySpan = (s, language) => {
+    const text = s.textContent
+    const ref = xrefGetMaybe(language == "reach" ? "rsh" : language, text);
```

Comment:
You should make a normalization dictionary and error if there's no normalization and/or consult the `aliases` field of Shiki's bundled languages (see the `sher` defn)